feature-libs/customer-ticketing/root/facade/customer-ticketing.facade.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { facadeFactory, QueryState } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { CUSTOMER_TICKETING_FEATURE } from '../feature-name';
+import { TicketDetails } from '../model';
+
+@Injectable({
+  providedIn: 'root',
+  useFactory: () =>
+    facadeFactory({
+      facade: CustomerTicketingFacade,
+      feature: CUSTOMER_TICKETING_FEATURE,
+      methods: ['getTicket'],
```

Comment:
need to add getTicketState as well 

make it the same order of it the same as it is in the abstract methods for my eyes 👁️ 👁️ 

-----
for further explanation on why we're doing this:
- you can lazy load a feature by calling one of it's methods as well instead of relying on the cms component to exist in a page.

Reply:
ohh, good to know. adding